assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook-admission
+  labels:
+    name: webhook-admission
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: webhook-admission
+  template:
+    metadata:
+      labels:
+        name: webhook-admission
+    spec:
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      containers:
+        - name: webhook-admission
+          securityContext:
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+            runAsUser: 65534
+            runAsGroup: 65534
+          image: "kinvolk/mutating-admission-controller:latest"
```

Comment:
Should have a versioned approach and not use the latest tag.

suggestion: change to v0.1.0 and also push the same tag to the image repository.

Reply:
I'd put a commit hash as a tag here. `v0.1.0` would be confusing with Lokomotive version.

Reply:
So it will of form `v0.3.0-72-g3ca8ab75-dirty` will this work? But this will change every time I make a change before merging of this PR and create problems. If you meant something else, can you please tell

Reply:
Also does the image work on arm machines ?
If not then we need to push both `amd64` and `arm64` in the tags and do the `docker manifest create/annotate/push` sequence as described [here](https://github.com/kinvolk/calico-hostendpoint-controller)

Reply:
Lokoctl is not building on `arm64` architecture, it get stuck in lokoctl build step

![Screenshot 2020-08-12 at 1 08 30 PM](https://user-images.githubusercontent.com/24803604/89988316-fd3f1480-dc9c-11ea-907a-5fda4bf7686e.png)

Reply:
Update: It got built, after a long time